README.md

```diff
@@ -54,6 +54,12 @@ As a final step, it's time to connect 2 envoy clients to `xds-relay`. You're goi
     envoy -c example/config-files/envoy-bootstrap-1.yaml # on the first window
     envoy -c example/config-files/envoy-bootstrap-2.yaml # on the second window
 
+Note: `envoy` here is a general reference to however you set up envoy binaries to run on your machine. If you are using 
+getenvoy to set up your envoy clients, be sure to first fetch the desired version. The commands to run would instead be:
+
+    getenvoy run <reference> -- -c example/config-files/envoy-bootstrap-1.yaml # on the first window
+    getenvoy run <reference> -- -c example/config-files/envoy-bootstrap-2.yaml # on the second window
+
```

Comment:
Would you be able to consolidate this paragraph, for example:

```
As a final step, it's time to connect 2 envoy clients to `xds-relay`. We will be using [getenvoy](link) to install the Envoy binaries. You're going to find 2 files named `envoy-bootstrap-1.yaml` and `envoy-bootstrap-2.yaml` that we're going to use to connect the envoy instances to `xds-relay`. Open 2 terminal windows and run:

    getenvoy run <reference> -- -c example/config-files/envoy-bootstrap-1.yaml # on the first window
    getenvoy run <reference> -- -c example/config-files/envoy-bootstrap-2.yaml # on the second window
```

Also what is `<reference>`?

Reply:
`<reference>` is the terminology used in [getenvoy's run command](https://www.getenvoy.io/reference/getenvoy_run/) to describe an envoy binary's manifest reference that was fetched beforehand. I can expand it to `<envoy binary reference/filepath>` if that makes it more clear. I just assumed this was standard terminology.